key_management/src/keystore.rs

```diff
@@ -147,18 +181,33 @@ impl KeyStore for MemKeyStore {
     fn remove(&mut self, key: String) -> Result<KeyInfo, Error> {
         self.key_info.remove(&key).ok_or(Error::KeyInfo)
     }
+
+    fn unlock(&mut self, _passphrase: &str) -> Result<(), EncryptedKeyStoreError> {
+        todo!()
```

Comment:
Did I start reviewing this too early? If not, just a reminder to fill this method.

Reply:
It's still a draft PR. Eyes on at any stage helps me out. And yes, this method will be used by the RPC methods and CLI commands later on, just haven't gotten that far yet.

Reply:
Ahh my bad, carry on